jsk_spotkinova_robot/README.md

```diff
@@ -97,26 +158,33 @@ If you want to send an angle-vector to kinova, you need to send the joint angles
 The first 12 dimensions are for spot, but they will be ignored and the last 6 dimensions will be sent to kinova only.
 ```
 (send *ri* :angle-vector #f(0.0 0.0 45.0 0.0 0.0 45.0 0.0 0.0 90.0 0.0 0.0 90.0 0.0 15.0 180.0 -130.0 0.0 55.0 90.0))
+
 ```
-To move the gripper 50 [mm] up, you can use `move-end-pos` method.
-```
-(send *spotkinova* :head :move-end-pos #f(0 0 -50))
-```
-You can also use `move-end-rot` method to turn the gripper.
+### Control the full body
+
+To change the full body pose, You can use `body-pose` method.
 ```
-(send *spotkinova* :head :move-end-rot -90 :z)
+(send *ri* :body-pose '(0 0.2 0))
+(send *ri* :body-pose '(0 -0.2 0))
+(send *ri* :body-pose '(0 0 0.2))
+(send *ri* :body-pose '(0 0 -0.2))
+(send *ri* :body-pose '(0.1 0 0))
+(send *ri* :body-pose '(-0.1 0 0))
```

Comment:
I am sorry, but could you please add information about coords?

Like

To change the full body pose, you can use `body-pose` method with rpy list or coords variable.
```
(send *ri* :body-pose '(0 0.2 0))
(send *ri* :body-pose '(0 -0.2 0))
(send *ri* :body-pose '(0 0 0.2))
(send *ri* :body-pose '(0 0 -0.2))
(send *ri* :body-pose '(0.1 0 0))
(send *ri* :body-pose '(-0.1 0 0))
(send *ri* :body-pose (make-coords :pos #f(0 0 50) :rpy #f(0.1 0 0)))
```

Reply:
Ok, I will add it.